### PR TITLE
Fix delete recording bug for renamed recordings

### PR DESCRIPTION
**Pull request by Alexjsenn**

### CODE REVIEW

**Comment by Alexjsenn** [comment]
Fixes #220

File: src/main/java/com/redhat/rhjmc/containerjfr/commands/Command.java [hunk 1]
```diff
@@ -71,6 +71,6 @@ default boolean validateTargetId(String targetId) {
     }
 
     default boolean validateRecordingName(String name) {
-        return name.matches("[\\w-_]+(\\.jfr)?");
+        return name.matches("[\\w-_]+(\\.\\d)?(\\.jfr)?");
```
Comment (andrewazores) [inline src/main/java/com/redhat/rhjmc/containerjfr/commands/Command.java:74]: What happens when this gets to double-digit rename suffixes?

File: src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/DeleteSavedRecordingCommandTest.java [hunk 2]
```diff
@@ -107,6 +107,12 @@ void shouldNotValidateIncorrectArgc(int argc) {
         MatcherAssert.assertThat(e.getMessage(), Matchers.equalTo(errorMessage));
     }
 
+    @Test
+    void shouldValidateRenamedRecording() {
```
Comment (andrewazores) [inline src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/DeleteSavedRecordingCommandTest.java:111]: Please make this a `@ParameterizedTest` and test various different numeric suffixes to be sure they all work.

File: src/main/java/com/redhat/rhjmc/containerjfr/commands/Command.java (hunk 1 above)
Comment (Alexjsenn) [reply src/main/java/com/redhat/rhjmc/containerjfr/commands/Command.java]: My bad, for some reason I had remembered wrong and thought it only renamed up to 9, it's fixed now.

**Pull request merged**